build-test.xml

```diff
@@ -62,6 +63,7 @@
             classpathref="classpath"
             verbose="false"
             debug="true"
+            debuglevel="lines,vars,source"
```

Comment:
We were actually setting -g:none, even though the -debug flag was set.  This tells exactly what debug info we want to add.  We don't need it for CI, but it's useful for regular development.